### ISSUE TITLE
Setup & Authorization Flow | Admin 2

### ISSUE THREAD

**Issue by bjackson27** [google/site-kit-wp#221]
Set up end to end testing for Admin 2 Site Kit activation.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* This must be an alternate scenario for the E2E test from #250, #251, and #360. As such, it is blocked by these to be completed first.
* The only piece that must be missing here is the client credentials step (they were already provided before).
* When initiating the setup flow, the user should immediately get to the "Sign in with Google" button (for the first admin it was step 2, here it should be step 1).
* The rest is similar to the original admin flow covered by the above issues.

## Implementation Brief

* N/A

## Changelog entry

* N/A

**Comment by felixarntz** [comment]
@bjackson27 @aaemnnosttv What is this regarding? Wouldn't the first admin "activate Site Kit"? In theory the first admin could deactivate Site Kit again after completing the setup, and then a second admin could reactivate it. This is a possible scenario, but if this issue here covers that case, it would be really low priority, as it's kind of trivial.

**Comment by aaemnnosttv** [comment]
@felixarntz I'm thinking this is about testing the setup flow with a second admin since it skips the client configuration.

**Comment by felixarntz** [comment]
Thanks @aaemnnosttv, updated the title (also those of similar issues) accordingly.

**Comment by lilybonney** [comment]
Moving this to the "Revisit Later" column per our team discussion yesterday (the work captured in the scope of this ticket will likely still be needed at some point, but cannot be executed as it stands right now).

**Comment by aaemnnosttv** [comment]
Just noting here that we discussed possibly removing this until later but this test was low effort to write and the PR addresses important stability issues related to changing users during the test.

**Comment by circlecube** [comment]
QA passes.

 PASS  tests/e2e/specs/auth-flow-admin-2.test.js (33.784s)
  the set up flow for the second administrator
    ✓ admin 2 (17903ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        33.841s, estimated 36s
Ran all test suites matching /tests\/e2e\/specs\/auth-flow-admin-2.test.js/i.
